### PR TITLE
TLT-2873: unicode course title support

### PR DESCRIPTION
**Pull request by elliottyates**
Please see https://jira.huit.harvard.edu/browse/TLT-2873.

### CODE REVIEW

File: web/views.py [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
+
+from .forms import IndexForm
+from .models import APIUser, School
 from canvas.apimethods import CanvasAPI, CanvasServiceException
-from canvas.apimodels import Term, SearchResults
 from mediasite.apimethods import MediasiteAPI, MediasiteServiceException
-from mediasite.apimodels import UserProfile, Role
-from .forms import IndexForm
-from .models import School, Log, APIUser
+from mediasite.apimodels import Role, UserProfile
```
Comment (sapnamysore) [inline web/views.py]: 👍  for cleaning up the imports

**Comment by sapnamysore** [comment]
Am getting an exception in trying to provision this course on dev '中文 ( ILE-elliott-test-zw )'. Discussed offline. Please fix.

**Comment by elliottyates** [comment]
test comment -- picked up by our integration?

**Comment by elliottyates** [comment]
This PR is superseded by #11. Closing this out-of-date PR and removing the branch.

**Pull request closed without merging**